mbld/libs.myr

```diff
@@ -146,6 +146,7 @@ const addlib = {b, sl, lib, added, diradded, looped
 	match std.htget(b.libs, lib)
 	| `std.None:
 		std.slpush(sl, std.fmt("-l{}", lib))
+		b.dynlink = true
```

Comment:
Maybe the right thing to do here is to return whether we're linking dynamically or statically from here.